javascript/brahma-firelight/app.ts

```diff
@@ -1,19 +1,45 @@
-import { createApp } from 'brahma-firelight';
+import { useBrahma, startServer } from "brahma-firelight";
 
-const app = createApp();
+useBrahma((req) => {
+    const { path } = req;
 
-// GET
-app.get('/', (req, res) => {
-  res.text('');
-});
+    // GET --> /
+    if (path === "/") {
```

Comment:
this seems to be an implementation tweaked for benchmark. the idea is to have a production-ready implementation to have the fairest comparison beetween each framework

Reply:
Sure, not saying that Brahma is not production ready, but using if to define routes (rather than express like API for routing) is not something expected to be written when we have an application to be delivered